cpp/src/community/k_truss_impl.cuh

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024-2025, NVIDIA CORPORATION.
+ * Copyright (c) 2024, NVIDIA CORPORATION.
```

Comment:
Looks like your copyright reverted... should include 2025.

Reply:
resolved